crates/autopilot/src/solvable_orders.rs

```diff
@@ -238,7 +234,19 @@ impl SolvableOrdersCache {
         });
 
         *self.cache.lock().unwrap() = Inner {
-            auction: Some(boundary::auction::to_domain(auction)),
+            auction: Some(domain::Auction {
```

Comment:
nit: Doesn't make a huge difference in this case because there is not a lot of contention on this lock but I think it's a good habit to prepare the data you want to store using the Mutex before you take the lock to keep the critical section as short as possible.

Reply:
Makes sense, extracted